a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -1316,7 +1313,7 @@ <h4>Examples</h4>
 						<li>
 							<span
 								data-localization-id="charts-diagrams-formulas-extended"
-								data-localization-mode="descriptive">Charts, diagrams, figures, and graphs have extended descriptions</span>
+								data-localization-mode="descriptive">Complex images are described by extended descriptions</span>
```

Comment:
A simple photography can be extendly described because of the book context or publisher will. 
I suggest we stick here with the minimum common denominator: images are described by extended descriptions
(I remember we had  a previous discussion about qualifying things as complex but can't find the reference)

Reply:
Something to consider. The same image say of the "Eiffel Tower" can just be a simple image with alt text "Eiffel Tower" but in an Structural Engineering textbook could have that same image but need a detailed description describing how the support beams are connecting, in this case the "simple image" becomes a "complex image" since it requires the extended description.  This is how I look at this anyway.  So it doesn't really matter how "simple" or how "complex" the image is just in our thinking that simple images that don't require additional information are simple and those that require or that the author feels they need to further describe with a longer description we classify as "complex" but this word can be whatever we wish to call any image having an additional description.